src/DbUtils.php

```diff
@@ -1609,7 +1609,7 @@ public function formatUserName($ID, $login, $realname, $firstname, $link = 1, $c
         if ($realname !== null && strlen($realname) > 0) {
             $formatted = $realname;
 
-            if (strlen($firstname) > 0) {
+            if ($firstname !== null && strlen($firstname) > 0) {
```

Comment:
Maybe more simple?
```suggestion
            if (strlen($firstname ?? '') > 0) {
```

Reply:
Why not, but I used the same style as L1609. Then I think I should change both to be the same. OK for you, @trasher  ?